src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/CopyNumberPosteriorLocatableRecord.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.hellbender.tools.copynumber.gcnv;
+
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.Map;
+
+/**
+ * A single copy number posterior record for a specific interval
+ */
+public class CopyNumberPosteriorLocatableRecord implements Locatable {
```

Comment:
Put this and `CopyNumberPosteriorRecord` in copynumber/formats/records.  I would probably also rename them as `CopyNumberPosterior` and `LocatableCopyNumberPosterior`, since none of the other records end in `Record`.

Reply:
Done